src/net_processing.cpp

```diff
@@ -3412,7 +3419,6 @@ void PeerManagerImpl::ProcessMessage(CNode& pfrom, const std::string& msg_type,
         if (!m_mempool)
```

Comment:
I think for brevity you can combine this check with the next one.

Reply:
Done. The earlier behaviour of the code was to disconnect from the peer who sent the compact block, and hence I thought it was necessary to have a dedicated log message accompanying it. This is not necessary anymore since the peer is not being disconnected.